src/devices/video/mc6847.cpp

```diff
@@ -1,4 +1,4 @@
-// license:BSD-3-Clause
+// license:BSD-3-Clause 
```

Comment:
You can avoid spurious changes like this by running `srcclean` on the file.

Reply:
Nice, ran the tool this time.